src/libraries/System.Private.Xml/src/System/Xml/XmlResolver.FileSystemResolver.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Threading.Tasks;
+
+namespace System.Xml
+{
+    public abstract partial class XmlResolver
+    {
+        /// <summary>
+        /// Gets an XML resolver which resolves only file system URIs.
+        /// </summary>
+        /// <value>An XML resolver which resolves only file system URIs.</value>
+        /// <remarks>
+        /// Calling <see cref="GetEntity"/> or <see cref="GetEntityAsync"/> on the
+        /// <see cref="XmlResolver"/> instance returned by this property will resolve only URIs which scheme is file.
+        /// </remarks>
+        public static XmlResolver FileSystemResolver => XmlFileSystemResolver.s_singleton;
+
+        // An XML resolver that resolves only file system URIs.
```

Comment:
Are XML files with file system references common enough? I do not think that I have ever seen file system references in XML files, except for maybe some tests. Would be better for the switch to disable resolving of any URLs?